e3sm_diags/driver/utils/regrid.py

```diff
@@ -16,11 +16,16 @@
 
```

Comment:
The code changes in this file involve updating the accepted hybrid sigma keys, including adding `"hyai"` and `"hybi"` which are used by the aerosol budget driver.